WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -321,6 +322,16 @@ private extension InPersonPaymentsMenuViewController {
             self?.tableView.reloadData()
         }.store(in: &cancellables)
     }
+
+    private func configureWebViewPresentation() {
+        viewModel.$showWebView.sink { viewModel in
+            guard let viewModel = viewModel else {
+                return
+            }
+            let connectionController = AuthenticatedWebViewController(viewModel: viewModel)
+            self.navigationController?.show(connectionController, sender: nil)
+        }.store(in: &cancellables)
+    }
```

Comment:
We can go through this bit deeper later in our pairing, but from what I understand: 

As we're showing the the AuthenticatedWebView here, we're subscribing and observing changes in the `AuthenticatedWebViewModel` state (changes in credentials, redirections, progressbar, etc, ...), and we use `.sink` to create this sort of subscription pipeline, which will end when our `receivedValue` (`AuthenticatedWebViewController`) is deallocated, is that right?